spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/HeaderKeys.java

```diff
@@ -0,0 +1,9 @@
+package org.springframework.cloud.sleuth.instrument.web;
+
+//Annotation key holding class for request and response headers
+abstract class HeaderKeys {
```

Comment:
HeaderKeys is a confusing name, especially in http, because a lot of these aren't headers. Moreover header isn't a conceptual name in tracing either.. maybe HttpKeys or something?